src/common/cluster-ipc.ts

```diff
@@ -128,4 +128,22 @@ if (ipcMain) {
 
     return Array.from(isAllowed);
   });
+
+  handleRequest(ClusterGlobalWatchChannel, async (event, clusterId: ClusterId): Promise<boolean> => {
```

Comment:
Not related to this PR but I think we need to consider moving `handleRequest` handlers to `main`. We could leave the channel names to `common` because they are used from both sides.

Reply:
Already done in https://github.com/lensapp/lens/pull/2821